solverdummy/solverdummy.py

```diff
@@ -19,8 +19,15 @@
 configuration_file_name = args.configurationFileName
 participant_name = args.participantName
 mesh_name = args.meshName
+if (participant_name == 'SolverOne'):
+  DataWrite_Name='Forces'
```

Comment:
```suggestion
  data_write_name = 'Forces'
```

Please use lowercase, with words separated by underscores for function and variable naming (see [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)).

Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not). See [here, second bulletpoint](https://www.python.org/dev/peps/pep-0008/#other-recommendations).